packages/Filter/stories/Hooks/Hooks.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import Filter, { useFilter } from "../../src";
+
+const columnsSettings = [
+  {
+    id: "goals",
+    type: Filter.types.columnTypes.NUMBER,
+    label: "Goals (NUMBER)",
+  },
+  {
+    id: "name",
+    type: Filter.types.columnTypes.TEXT,
+    label: "Name (TEXT)",
+  },
+  {
+    id: "joined",
+    type: Filter.types.columnTypes.DATE,
+    label: "Joined by (DATE)",
+    momentParsingFormat: "MM/DD/YYYY",
+  },
+  {
+    id: "shareable",
+    type: Filter.types.columnTypes.BOOLEAN,
+    label: "Shareable (BOOLEAN)",
+  },
+  {
+    id: "position",
+    type: Filter.types.columnTypes.SINGLE_SELECT,
+    label: "Position (SINGLE SELECT)",
+  },
+];
+
+export default function FilterWithServer() {
+  const {
+    filters,
+    // filteredData,
```

Comment:
I think we better show `filteredData` in this story, since it demonstrates with `useFilter` hook, filter logic is implemented for you

Reply:
Ok i have added a bit about filtered data. I dont think I want to actually render it in this example; I'm trying to keep it as simple as possible.  They could look at the `MockServer` or `ShowCase` example to see how that works.